### PR TITLE
Lesson 3 assignments 1 thru 3

### PR DESCRIPTION
**Pull request by cdelapena**
Pull request created after completion of Lesson 3 assignments 1 thru 3.

### CODE REVIEW

File: Lesson03/exercise-one.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+console.log(lionKing);
+
+lionKing.splice(2, 1);
+
+console.log(lionKing);
+
+let lionKingString = lionKing.join(',')
+
+console.log(lionKingString);
```
Comment (MadEste) [inline Lesson03/exercise-one.js]: 👍 Perfect!

File: Lesson03/exercise-two.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var sentence = "Oh my it is a lovely day.";
+
+var sentenceArray = sentence.split(" ");
+
+console.log(sentenceArray[2]);
```
Comment (MadEste) [inline Lesson03/exercise-two.js]: 👍

File: Lesson03/exercise-three.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+let colors = ['violet', 'blue', 'green', 'yellow', 'orange', 'red'];
+
+for (let i = 0; i < colors.length; i++) {
+
+  console.log(i + ',' + colors[i]);
```
Comment (MadEste) [inline Lesson03/exercise-three.js]: 👍 Good. 
You can also use template literals in javascript for String concatenation:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

**Comment by cdelapena** [comment]
Closing assignment 3 pull request in preparation for assignment 4 pull request.

**Pull request closed without merging**